src/workerd/io/compatibility-date.capnp

```diff
@@ -254,4 +254,10 @@ struct CompatibilityFlags @0x8f8c1b68151b6cef {
   # WHATWG URL Standard, leading to a number of issues reported by users. Unfortunately,
   # the specCompliantUrl flag did not contemplate the redirect usage. This flag is
   # specifically about the usage in a redirect().
+
+  actorMapEvictionShorterTimeout @24 :Bool
+      $compatEnableDate("2023-01-24");
```

Comment:
Compat enable dates must always be in the future, such that support for the flag has been fully deployed to production before the flag becomes default-on.

You need to define `$compatEnableFlag` and `$compatDisableFlag`.